doc/release-notes/5.9-release-notes.md

```diff
@@ -0,0 +1,168 @@
+- Users will be able to use a new role to allow a user to respond to file download requests without also giving them the power to manage the dataset  (Issue #8109, PR #8174)
+- Users will no longer be forced to update their passwords when moving from Dataverse 3.x to Dataverse 4.x (PR #7916)
+- Improved accessibility of buttons on the Dataset and File pages (Issue #8247, PR #8257)
+
+## Notes for Dataverse Installation Administrators
+
+### Indexing Performance on Datasets with Large Numbers of Files
+
+We discovered that whenever a full reindexing needs to be performed, datasets with large numbers of files take an exceptionally long time to index. For example, in the Harvard Dataverse Repository, it takes several hours for a dataset that has 25,000 files. In situations where the Solr index needs to be erased and rebuilt from scratch (such as a Solr version upgrade, or a corrupt index, etc.) this can significantly delay the repopulation of the search catalog.
+
+We are still investigating the reasons behind this performance issue. For now, even though some improvements have been made, a dataset with thousands of files is still going to take a long time to index. In this release, we've made a simple change to the reindexing process, to index any such datasets at the very end of the batch, after all the datasets with fewer files have been reindexed. This does not improve the overall reindexing time, but will repopulate the bulk of the search index much faster for the users of the installation.
+
+### Custom Analytics Code Changes
+
+You should update your custom analytics code to capture a bug fix related to tracking within the dataset files table. This release restores that tracking.
+
+For more information, see the documentation and sample analytics code snippet provided in [Installation Guide](http://guides.dataverse.org/en/5.9/installation/config.html#web-analytics-code). This update can be used on any version 5.4+.
+
+### New ManageFilePermissions Permission
+
+Dataverse can now support a use case in which a Admin or Curator would like to delegate the ability to grant access to restricted files to other users. This can be implemented by creating a custom role (e.g. DownloadApprover) that has the new ManageFilePermissions permission. This release introduces the new permission, and a Flyway script adjusts the existing Admin and Curator roles so they continue to have the ability to grant file download requrests.
+
+### Thumbnail Defaults
+
+New defaults have been added for when to create thumbnails for images and PDFs. The default is 3 MB for images, 1 MB for PDFs. Previously, there was no default and the thumbnail creation process for large files had the potential to cause system stability issues. This change will not affect your installation if you already have your own limits specified (via JVM options in domain.xml).
```

Comment:
No, sorry, can we have this instead (instead of this whole paragraph):

"New *default* values have been added for the JVM settings ``dataverse.dataAccess.thumbnail.image.limit`` and ``dataverse.dataAccess.pdf.image.limit``, of 3MB and 1MB respectively. This means that, *unless specified otherwise* by the JVM settings already in your domain configuration, the application will skip attempting to generate thumbnails for image files and PDFs that are above these size limits. 
In previous versions, if these were not explicitly set, the application would try to create thumbnails for files of any size. Which would occasionally result in problems with very large images. "

Reply:
(I just edited the comment above a couple of times, to preserve the formatting, back ticks and such...)

Reply:
Sure thing, https://github.com/IQSS/dataverse/pull/8284/commits/b498b119f401076f140bb793a58b0b0090998443